src-vue/src/plugins/store.ts

```diff
@@ -382,6 +388,16 @@ async function _initializeApp(state: any) {
         // Check installed Northstar version if found
         await _get_northstar_version_number(state);
     }
+
+    await invoke("get_server_player_count")
+        .then((message) => {
+            state.player_count = (message as any)[0];
+            state.server_count = (message as any)[1];
+        })
```

Comment:
Here's the declaration of the `invoke` method:
```typescript
declare function invoke<T>(cmd: string, args?: InvokeArgs): Promise<T>;
```

The way you're calling it right now (`invoke("get_server_player_count")`) doesn't set the `T` type, so returned promise will itself return an `unknown`, which prevents you to use return value as you would like to.

To fix this, since you know this `invoke` call will return an array of two number, you can summon it with according type:
```typescript
await invoke<[number, number]>("get_server_player_count")
        .then((message) => {
            state.player_count = message[0];
            state.server_count = message[1];
        });
```

Reply:
Thanks for the detailed explanation <3

Done in a8b0f44747afcdd880f6d1ce2d8a2b350ead4311